novawallet/Modules/BackupAttention/BackupAttentionViewLayout.swift

```diff
@@ -1,6 +1,34 @@
 import UIKit
+import SoraUI
 
 final class BackupAttentionViewLayout: UIView {
```

Comment:
Let's inherit here from ScrollableContainerLayoutView and use stackView.layoutMargins to control bottom offset like it is done in StartStakingInfoViewLayout